indico/modules/events/notifications.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from sqlalchemy.orm import joinedload
+
+from indico.core.config import Config
+from indico.core.notifications import send_email, make_email
+from indico.modules.categories.models.categories import Category
+from indico.web.flask.templating import get_template_module
+
+
+def notify_event_creation(event, following_events=None):
+    config = Config.getInstance()
+    addrs = set()
+    chair = '; '.join(x.full_name for x in event.person_links)
+    category = event.category
+    query = category.chain_query.filter(Category.notify_managers).options(joinedload('acl_entries'))
+    following_events = following_events or []
```

Comment:
This is a bit out of place here - if it's not needed in the template let's avoid it, and otherwise let's do it before rendering the template. But not between the query and iterating over it.